eng/Versions.props

```diff
@@ -129,11 +129,11 @@
     <MicrosoftVisualStudioShellInterop80Version>8.0.50728</MicrosoftVisualStudioShellInterop80Version>
     <MicrosoftVisualStudioTelemetryVersion>16.0.233</MicrosoftVisualStudioTelemetryVersion>
     <MicrosoftVisualStudioTemplateWizardInterfaceVersion>8.0.0.0-alpha</MicrosoftVisualStudioTemplateWizardInterfaceVersion>
-    <MicrosoftVisualStudioTextDataVersion>16.1.101</MicrosoftVisualStudioTextDataVersion>
-    <MicrosoftVisualStudioTextInternalVersion>16.1.101</MicrosoftVisualStudioTextInternalVersion>
-    <MicrosoftVisualStudioTextLogicVersion>16.1.101</MicrosoftVisualStudioTextLogicVersion>
-    <MicrosoftVisualStudioTextUIVersion>16.1.101</MicrosoftVisualStudioTextUIVersion>
-    <MicrosoftVisualStudioTextUIWpfVersion>16.1.101</MicrosoftVisualStudioTextUIWpfVersion>
+    <MicrosoftVisualStudioTextDataVersion>16.2.35-g84aa2010de</MicrosoftVisualStudioTextDataVersion>
+    <MicrosoftVisualStudioTextInternalVersion>16.2.35-g84aa2010de</MicrosoftVisualStudioTextInternalVersion>
+    <MicrosoftVisualStudioTextLogicVersion>16.2.35-g84aa2010de</MicrosoftVisualStudioTextLogicVersion>
+    <MicrosoftVisualStudioTextUIVersion>16.2.35-g84aa2010de</MicrosoftVisualStudioTextUIVersion>
+    <MicrosoftVisualStudioTextUIWpfVersion>16.2.35-g84aa2010de</MicrosoftVisualStudioTextUIWpfVersion>
```

Comment:
I'd prefer to separate the VSSDK update from the code update.

Maybe need to confirm with @amwieczo@microsoft.com  that we can use this update is public code.

Reply:
Will send out separate PR once the editor change is merged and official package is available

Reply:
Will wait for it.

---
In reply to: [296015268](https://github.com/dotnet/roslyn/pull/36146#discussion_r296015268) [](ancestors = 296015268)

Reply:
PR for editor SDK upgrade: 
https://github.com/dotnet/roslyn/pull/36811